src/@lekoarts/gatsby-theme-minimal-blog/components/post.tsx

```diff
@@ -49,9 +49,27 @@ const Post: React.FC<React.PropsWithChildren<PageProps<MBPostProps>>> = ({
   const ns = post?.ns;
   const isTranslationEnabled = !!ns;
   const { t } = useTranslation(ns);
+  const firstTag = post.tags && post.tags[0];
+  const isBookReview = firstTag && firstTag.name === 'Обзор книги';
+
+  const title = isBookReview
+    ? `${t('brief_content')} - ${t('title')}`
+    : post.title;
+
+  const description = isBookReview
```

Comment:
Author's name is missing; Its specified in the `description` field of the .mdx file header;
![Screenshot by Dropbox Capture](https://github.com/kaleemullahdev/romanrud-blog/assets/12322543/defb47f4-b102-41fe-8240-20d5d0db840d)